uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -420,15 +381,18 @@ public static UiObject obtainUiObjectWithExactText(@NonNull final String text) {
      */
     public static UiObject obtainUiObjectWithClassAndIndex(@NonNull final Class clazz, final int index) {
         Logger.i(TAG, "Obtain an instance of the UiObject for the class name:" + clazz + " and index value:" + index);
+        return obtainUiObjectWithUiSelector(new UiSelector().className(clazz).index(index),
+                FIND_UI_ELEMENT_TIMEOUT);
+    }
+
+    public static UiObject obtainUiObjectWithUiSelector(@NonNull final UiSelector selector, final long existsTimeout) {
```

Comment:
why did we pass exists timeout here? Can we not directly use FIND_UI_ELEMENT_TIMEOUT here?